lib/actions/sendTemplate.js

```diff
@@ -83,24 +92,8 @@ function createParams(msg, cfg) {
     };
 }
 
-function checkResult(response) {
-    if (!response.length) {
-        return null;
-    }
-
-    const result = response[0];
-
-    console.log('Sending status:', result.status);
-
-    if (result.status === 'rejected' || result.status === 'invalid') {
-        throw new Error(JSON.stringify(result));
-    }
-
-    return result;
-}
-
 function getTemplateModel(cfg) {
-    console.log('About to retrieve available templates');
+    logger.info('About to retrieve available templates');
```

Comment:
Why logger instead of this.logger?

Reply:
It's metadata uploader. No difference here